Linear_Search/Linear_Search.rs

```diff
@@ -0,0 +1,16 @@
+fn linear_search<T: Ord>(vec: &Vec<T>, key: &T) -> Option<usize> {  
```

Comment:
This file should not be a part of this pr.

Reply:
Resolved this issue & updated my PR branch.